tools/dataflow-throttling/src/main/java/com/google/cloud/pso/dataflowthrottling/ThrottlingException.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.pso.dataflowthrottling;
+
+/**
+ * This class processes the exception which is raised by clientCall.
```

Comment:
This exception must be thrown by the clientCall method when the external service returns an error because it throttled the request.